apps/challenges/models.py

```diff
@@ -575,6 +575,9 @@ class ChallengeEvaluationCluster(TimeStampedModel):
         max_length=512, null=True, blank=True
     )
     route_table_id = models.CharField(max_length=512, null=True, blank=True)
+    route_table_association_ids = ArrayField(
```

Comment:
@savish28 Can you add a brief comment here as to what this is for?

Reply:
In the cluster setup when we create route table we associate them 

> @savish28 The code in `aws_utils` seems to have many levels (aka recursive?) of calls inside the delete function. Seems a bit redundant and maybe you can flatten this by having each function delete what is described in the name, and nothing else. So max 2 level of callbacks. More readable that way. (cluster->challenge->challenge specific aws entity)
> 
> Let me know if you had any specific reason for doing these redirections in each of the methods.

@KhalidRmb Basically I have tried to replicate the same pipeline for cluster creation. The creation pipeline uses same functions so it for better abstraction and similarity in code I have followed this function abstraction.

Reply:
Updated as a comment in the code